crates/iota-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -2660,29 +2659,33 @@ impl AuthorityPerEpochStore {
                 checkpoint_roots.push(consensus_commit_prologue_root);
             }
             checkpoint_roots.extend(roots.into_iter());
+
+            if let Some(randomness_round) = randomness_round {
+                randomness_roots.insert(TransactionKey::RandomnessRound(
+                    self.epoch(),
+                    randomness_round,
+                ));
+            }
+            // Determine whether to write pending checkpoint for user tx with randomness.
```

Comment:
upstream had a newline between the bracket and the comment. Why did you remove it?

Reply:
Added the newline back in [a8a6d2a](https://github.com/iotaledger/iota/pull/4899/commits/a8a6d2a8e0acbbd52da53e377d7559971bb1c5a1), thanks for pointing that out!